doc/examples/applications/plot_fluorescence_nuclear_envelope.py

```diff
@@ -0,0 +1,214 @@
+"""
+======================================================
+Measure fluorescence intensity at the nuclear envelope
+======================================================
+
+This example reproduces a well-established workflow in bioimage data analysis
+for measuring the fluorescence intensity localized to the nuclear envelope, in
+a time sequence of cell images (each with two channels and two spatial
+dimensions) which shows a process of protein re-localization from the
+cytoplasmic area to the nuclear envelope. This biological application was
+first presented by Andrea Boni and collaborators in [1]_; it was used in a
+textbook by Kota Miura [2]_ as well as in other works ([3]_, [4]_).
+In other words, we port this workflow from ImageJ Macro to scikit-image.
+
+.. [1] Boni A, Politi AZ, Strnad P, Xiang W, Hossain MJ, Ellenberg J (2015)
+       "Live imaging and modeling of inner nuclear membrane targeting reveals
+       its molecular requirements in mammalian cells" J Cell Biol
+       209(5):705–720. ISSN: 0021-9525.
+       :DOI:`10.1083/jcb.201409133`
+.. [2] Miura K (2020) "Measurements of Intensity Dynamics at the Periphery of
+       the Nucleus" in: Miura K, Sladoje N (eds) Bioimage Data Analysis
+       Workflows. Learning Materials in Biosciences. Springer, Cham.
+       :DOI:`10.1007/978-3-030-22386-1_2`
+.. [3] Klemm A (2020) "ImageJ/Fiji Macro Language" NEUBIAS Academy Online
+       Course: https://www.youtube.com/watch?v=o8tfkdcd3DA
+.. [4] Vorkel D and Haase R (2020) "GPU-accelerating ImageJ Macro image
+       processing workflows using CLIJ" https://arxiv.org/abs/2008.11799
+
+"""
+
+import imageio
+import matplotlib.pyplot as plt
+import numpy as np
+import plotly.io
+import plotly.express as px
+from scipy import ndimage as ndi
+
+from skimage import (
+    filters, measure, morphology, segmentation
+)
+
+
+#####################################################################
+# We start with a single cell/nucleus to construct the workflow.
+
+dat = imageio.volread('http://cmci.embl.de/sampleimages/NPCsingleNucleus.tif')
+
+print(f'shape: {dat.shape}')
+
+#####################################################################
+# The dataset is a 2D image stack with 15 frames (time points) and 2 channels.
+
+fig = px.imshow(
+    dat,
+    facet_col=1,
+    animation_frame=0,
+    binary_string=True,
+    labels={'animation_frame': 'time point', 'facet_col': 'channel'}
+)
+plotly.io.show(fig)
+
+#####################################################################
+# To begin with, let us consider the first channel of the first image.
+
+ch0t0 = dat[0, 0, :, :]
+
+#####################################################################
+# Segment the nucleus rim
+# =======================
+
+smooth = filters.gaussian(ch0t0, sigma=2)
+
+thresh = smooth > 0.1
+
+fill = ndi.binary_fill_holes(thresh)
```

Comment:
Also this step is not in [the original workflow](https://github.com/NEUBIAS/neubias-springer-book-2020/blob/master/Ch2%20Measurements%20of%20Intensity%20Dynamics%20at%20the%20Periphery%20of%20the%20Nucleus/Code/code_final.ijm#L33). I could imagine that it's not necessary if one uses Otsu's thresholding method...

Reply:
Right, well, this was always a bit of a mystery to me... Looking at [steps c) and d)](https://link.springer.com/chapter/10.1007%2F978-3-030-22386-1_2#Fig3), I've wondered how the small holes were handled: Does dilation automatically take care of filling them?

I don't see what difference using Otsu's thresholding method would make (see above https://github.com/scikit-image/scikit-image/pull/5262#discussion_r713413342)... Even with multi-Otsu thresholding (defaulting to 3 classes), and keeping the lower threshold, there are still small holes inside the main region:
![multi_otsu](https://user-images.githubusercontent.com/2227806/134247429-3bcd1a65-3eac-4eb0-ad37-79110bf8868b.png)